### ISSUE TITLE
old podspec

### ISSUE THREAD

**Issue by usmayoung** [Xenofex/YAJL-Entity#4]
Hi @Xenofex , 

I hope all is going well.  I noticed you maintained this code base but have not pushed the latest podspec; therefore, I'm unable to pull your latest code.  Any chance you can push your pod spec repo to reflect the latest branch version.

**Comment by Xenofex** [comment]
Hello @usmayoung! Long time no see! The thing is, Cocoapods spec check prevents me from doing so. I always complains that I had a bug somewhere but I wasn't able to figure it out. And I already tested that this repo works fine by adding it from a git repostory (copied from my current project):

    pod 'YAJL-Entity', git: "https://github.com/Xenofex/YAJL-Entity.git", tag: "v1.1"

Hope that helps!

**Comment by usmayoung** [comment]
yeah @Xenofex , I think we have the same problem. I'm trying to split my project into pods for some features; therefore, I need to have 'YAJL-Entity' as dependency in my Podspec, but based on podspec validation I can't push with the "git: , tag:" format.  I'm only allowed the normal pod 'name, and version as defined in that podspec.  

Thanks for your help anyway.

**Comment by Xenofex** [comment]
Recently Cocoapods has updated to version 1.0.0. I may try to push again

**Comment by Xenofex** [comment]
@usmayoung I already tried Cocoapods 1.0.0, same error. Still not able to push to trunk.
